ui/component/core/src/index.ts

```diff
@@ -835,7 +837,7 @@ export class Manager implements EventProviderFactory {
                 authenticated = true;
 
                 // Get user roles
-                const rolesResponse = await rest.api.UserResource.getCurrentUserRoles();
+                const rolesResponse = await rest.api.UserResource.getCurrentUserClientRoles(OPENREMOTE_CLIENT_ID);
```

Comment:
Replace `OPENREMOTE_CLIENT_ID` with `this._config.clientId || OPENREMOTE_CLIENT_ID`

Reply:
Switched to use `this._config.clientId` as `normaliseConfig` will take care to set this